README.md

```diff
@@ -1,2 +1,63 @@
 # bugzilla_to_jira
 Python script to migrate your Bugzilla ticket to Jira
+
+### Installation
+
+Clone the repo, or download the raw bugToJira.py
```

Comment:
It seems you need jira python module. I supposed it is not native. An indication about the pip dependency to install would be nice.